lifelines/estimation.py

```diff
@@ -29,19 +31,109 @@ def __repr__(self):
         return s
 
 
-class ExponentialFitter(BaseFitter):
+class WeibullFitter(BaseFitter):
+
     """
-    This class implements an exponential model for univariate data. 
+    This class implements an exponential model for univariate data.
```

Comment:
Shouldn't this say "...a Weibull model..." ? Or perhaps a sentence on how it differs from the one named "ExponentialFitter"?